src/pytest_dbfixtures/factories/postgresql.py

```diff
@@ -210,17 +210,18 @@ def stop_server_and_remove_directory():
     return postgresql_proc_fixture
 
 
-def postgresql(process_fixture_name, db=None):
+def postgresql(process_fixture_name, db=None, scope='function'):
     """
     postgresql database factory.
 
     :param str process_fixture_name: name of the process fixture
-    :param int db: database name
+    :param str db: database name
+    :param str scope: scope of database fixture
```

Comment:
function should be the default pytest's value for fixtures without having defined scope.